scenes/enumerated/problem_12723/english_text.py

```diff
@@ -0,0 +1,9 @@
+from manim import DOWN, LEFT
+
+task_number_str = "P. 12723"
+air_mass_str = " grams of air"
+molecules_in_one_gram_str = r"in $1$ gram of air are $2 \cdot 10^{22}$ molecules"
```

Comment:
երևի հետևյալը ավելի լավն է․
"There are $2 \cdot 10^{22}$ molecules in $1$ gram of air"